usaspending_api/etl/management/commands/load_query_to_delta.py

```diff
@@ -50,6 +56,20 @@
         "custom_schema": "recipient_hash STRING, federal_accounts STRING, cfdas ARRAY<STRING>,"
         " tas_components ARRAY<STRING>",
     },
+    "recipient_profile": {
+        "model": RecipientProfile,
+        "source_query": recipient_profile_load_sql_string,
```

Comment:
Could you make this plural to give a hint to the reader that this is a list of strings?

Reply:
sure